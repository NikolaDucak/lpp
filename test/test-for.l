// for
for(var a = 5; a > 10; a = a+1){
	println("shouldn't be printent");
}

for(var a = 0; a < 5; a = a+1){
	println(a);
}

for(a =10; a < 40; a = a+10)
	println(a);

var a = 0;
for(; a < 3.5; a = a+0.5)
	println(a);


a = 1;
for(; a<9;){
	println(a);
	a = a+1;
}

for(; false; ){
	println("shouldn't be printent");
}
