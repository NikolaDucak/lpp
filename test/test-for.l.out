0
1
2
3
4
10
20
30
0
0.5
1
1.5
2
2.5
3
1
2
3
4
5
6
7
8
